reviewing/plugin-commands-listing/src/list.ts

```diff
@@ -91,6 +92,10 @@ For options that may be used with `-r`, see "pnpm help recursive"',
             name: '--dev',
             shortAlias: '-D',
           },
+          {
+            description: 'Display only dependencies that are also projects within the monorepo.',
+            name: '--only-projects',
```

Comment:
This breaks the convention of the existing `--dev` and `--prod` flags a bit. I think it's more clear what `pnpm list --only-projects` is doing as opposed to `pnpm list --projects`, but can update this if simply `--projects` is preferred.